File: guanabara-java/src/com/danillo/algoritmos/lista01/Exercicio03.java
package com.danillo.algoritmos.lista01;

import java.util.Scanner;

public class Exercicio03 {
    public static void main(String[] args){
        Scanner entrada = new Scanner(System.in);

        System.out.println("Qual é o nome do funcionário?");
        String nome = entrada.nextLine();
        System.out.println("Qual é o salário do funcionário?");
        float salario = entrada.nextFloat();
        System.out.println("Nome: " +nome+ "\nSalário: R$ " +salario);
    }
}

File: guanabara-c/lista01/exercicio03.c
#include <stdio.h>

int main(){
    char nome[30];
    float salario;

    printf("Insira o nome: ");
    scanf("%s", &nome);
    printf("Insira o salário: ");
    scanf("%f", &salario);
    printf("O funcionário %s tem um salário de R$ %.2f", nome, salario);

    return 0;
}
